core/shared/src/main/scala/spire/std/float.scala

```diff
@@ -6,7 +6,7 @@ import spire.math.Rational
 
 import java.lang.Math
 
-trait FloatIsField extends Field[Float] {
+trait FloatIsField extends Field.WithDefaultGCD[Float] {
```

Comment:
I think the existing gcd/lcm impl does some fancy stuff, but more matched the field of fractions impl instead. Any reason to prefer this one? I wrote the last one, so have some attachment, but definitely happy to see it go if it doesn't make sense anymore.